test/unittests/mongoBackend/mongoContextProvidersUpdate_test.cpp

```diff
@@ -523,20 +523,27 @@ TEST(mongoContextProvidersUpdateRequest, noPatternAttrOneSingle)
   EXPECT_EQ(SccOk, ms);
 
   EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
Yes, let's fix only what is 'touched' in this PR.
The rest, let's create an issue about it